src/AWS.Deploy.Common/TypeHintData/FilePathTypeHintData.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace AWS.Deploy.Common.TypeHintData
+{
+    /// <summary>
+    /// Additional typehint data for file options
+    /// </summary>
+    public class FilePathTypeHintData
+    {
+        /// <summary>
+        /// Corresponds to a Filter property for a System.Windows.Forms.FileDialog
+        /// to determine the choices that would appear in the dialog box if a wrapping tool prompted the user for a file path via a UI.
+        public string Filter { get; set; } = "All files (*.*)|*.*";
+
+        /// <summary>
+        /// Corresponds to the DefaultExt property for a System.Windows.Forms.FileDialog
+        /// to specify the default extension used if the user specifies a file name
+        /// without an extension.
+        /// </summary>
+        public string DefaultExtension { get; set; } = "";
```

Comment:
This property doesn't seem to be defined in TypeHintData for FilePath in src/AWS.Deploy.Recipes/DeploymentBundleDefinitions/Container.deploymentbundle in this CR.  Should it be?